docs/deployment.md

```diff
@@ -22,15 +22,79 @@ Dependencies:
 - [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
 - [helm](https://helm.sh/docs/intro/install/)
 
-TODO: Describe helm charts and how to install them on kubernetes cluster once we have them
+Rafiki cannot be run by itself but needs at least a Postgres and a Redis instance running with it. If you prefer to use Tigerbeetle instead of Postgres for accounting, a Tigerbeetle instance is required as well.
+
+An example Chart including Rafiki, Postgres, and Redis can be found [here](../infrastructure/helm/rafiki-example/).
+
+<!-- TODO: add chart to packages -->
+
+<!-- To add the Rafiki chart, run
 
 ```
 // add rafiki repository
 $ helm repo add ...
 
-// install rafiki components
-$ helm install ...
-```
+``` -->
```

Comment:
I still need to figure out where we host the actual charts. This is why this is commented out. Github does to offer a chart repository. 

I'll add that information in a future PR once I've figured out where to host the chart.